CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the OpenColorIO Project. -->
 
+**Version 2.0.0 (unreleased):**
+    * Renamed sanityCheck() method to validate() in the classes
+      Config, FileRule and ViewingRule
+    * Renamed Sanity enumeration to Validation in Config::Impl class,
+      and its members SANITY_UNKNOWN, SANITY_SANE and SANITY_INSANE to
+      VALIDATION_UNKNOWN, VALIDATION_VALID and VALIDATION_INVALID
```

Comment:
The enum doesn't seem to be used outside the class, thus I'm unsure if it should be mentioned at all.

Reply:
Looking at the rest of the changelog, it seems to describe changes from a user perspective. Hence, I simplified it to a single changelog entry. Removed the mention of the C++ classes, as it equally applies to the Java and Python interfaces.